script/chart_sync_utils.sh

```diff
@@ -170,11 +170,11 @@ commitAndSendExternalPR() {
     git add --all .
     git commit -m "kubeapps: bump chart version to $chartVersion"
     # NOTE: This expects to have a loaded SSH key
-    if [[ -n "$(git ls-remote origin $targetBranch  | wc -l)" ]]; then
-     echo "The remote branch '$targetBranch' already exists, please check if there is already an open PR at the repository '${CHARTS_REPO_ORIGINAL}'"
-    else
+    if [[ $(git ls-remote origin $targetBranch  | wc -l) -eq 0 ]] ; then
```

Comment:
Not sure why this file has come into the PR

Reply:
Idem